contrib/bubblejail/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "bubblejail"
+pkgver = "0.8.2"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "desktop-file-utils",
+    "libseccomp-devel",
+    "python-jinja2",
+    "python-tomli-w",
+    "python-pyxdg",
+]
+depends = [
+    "desktop-file-utils",
+    "libseccomp-devel",
```

Comment:
this is wrong, should probably just be removed

Reply:
checked again- it uses ctypes.util.find_library("seccomp")- you only need 'libseccomp' (non-devel)

Reply:
and you need that in depends

Reply:
also, you need xdg-dbus-proxy in depends